Create pwa-features-2020-comparison.md (#1135)
simplabs/simplabs.github.io: _posts/2020-06-05-pwa-features-2020-comparison.md

---

> title: 'PWA support on mobile and desktop in 2020'

Reviewer: ```suggestion
title: 'The state of PWA support on mobile and desktop in 2020'
```

context. See the table above for the full list.

On the desktop, native hardware features are mostly limited to the harddrive,
camera, and microphone as well as geolocation. Like any browser or app, it neatly
prompts the user to give or block access to the hardware. There just aren't that
many native sensors in a desktop device, except hardware that plugs directly
into the OS environment (e.g. bluetooth headphones, hard drives, or USB devices).

### What native software features are accessible by PWAs?

For a PWA to be fully indistinguishable from a native app, it needs to blend in

> with the native user interactions offered by the operating software. For example, native push

Reviewer: ```suggestion
with the native user interactions offered by the operating system. For example, native push
```

Each OS has different users. Designers, developers and the youth use macOS,
while employees of many corporates spend most of their time on Windows PCs. iOS
is not as big as Android. Only in the United States, more than 20% of the online population

> is on an iPhone or iPad. So it really depends on what audience you try to serve.

Reviewer: ```suggestion
are on iPhones or iPads. So it really depends on what audience you try to serve.
```

To me that sounds better than referring to singular devices.
